courses/CEIR11.md

```diff
@@ -0,0 +1,74 @@
+---
+code: CEIR11
+title: Engineering Mechanics
+
+specifics:
+  - branch: CE
+    semester: 1
+    credits: [3, 0, 0, 3]
+
+prereq: [Knowledge of Applied Physics]
```

Comment:
Prereq should only contain course codes; it can't have a phrase. Replace `Knowledge of Applied Physics` with `PHIR11`